### DISCUSSION TITLE
How to give friend access to push to my repo without giving him my account?

### DISCUSSION THREAD

**Discussion by TechyGeek55** [community/community#126115] [New to GitHub] [answered]
As title states

**Comment by m4xy07** [comment, marked as answer]
You can go to the repo settings, and invite the user as an collaborator. That way they will gain push access without the need for your account.

**Reply by Masterart666** [reply]
สวัสดี

**Comment by Sentini14** [comment]
I will definitely give out my time to guide you on how to switch and set up a new recommended QFS Ledger and start stocking up as little as you earn

**Comment by Prabhakar-Naik** [comment]
1.You can give the ssh key for access for your account.
2.You can go to the repo settings, and invite the user as a collaborator.
